library/cwm/src/lib/cwm/wrapper_widget.rb

```diff
@@ -14,11 +14,12 @@ module CWM
   #   )
   class WrapperWidget < AbstractWidget
     # Creates new instance with specified id and content
-    # @param widget_id [String] name of widget used as identified, have to be unique.
+    # @param id [String] name of widget used as identified, have to be unique.
     #   It have to be same as real widget id in content, otherwise enable/disable won't work.
-    # @param content [Hash<String, Object>] CWM hash definition
-    def initialize(widget_id, content)
-      self.widget_id = widget_id
+    #   If nil is used, it use default widget_id from class name.
+    # @param content [CWM::StringTerm] CWM hash definition
```

Comment:
No, a widget **hash** is not the same thing as a widget **term**, or a string term! I even linked to the thing so I will link again.
http://www.rubydoc.info/github/yast/yast-yast2/CWM/WidgetHash <- this is wanted here, because cwm_definition uses it
http://www.rubydoc.info/github/yast/yast-yast2/CWM/WidgetTerm
http://www.rubydoc.info/github/yast/yast-yast2/CWM/StringTerm